espnet/asr/pytorch_backend/asr.py

```diff
@@ -609,11 +629,23 @@ def enhance(args):
     # load trained model parameters
     logging.info('reading model parameters from ' + args.model)
     model_class = dynamic_import(train_args.model_module)
-    model = model_class(idim, odim, train_args)
-    assert isinstance(model, ASRInterface)
-    torch_load(args.model, model)
+    if train_args.frontend_module is not None:
+        # FIXME(kamo): Assuming fbank feature, so you need to change with the other frontend type
```

Comment:
remove this if block. FrontendASR can set its own `idim` by `train_args`

Reply:
I can't understand out what you mean. This block is inevitable.